app/src/main/kotlin/com/tailoredapps/androidapptemplate/MyApp.kt

```diff
@@ -40,12 +40,12 @@ class MyApp : Application() {
         Timber.plant(Timber.DebugTree())
         AndroidThreeTen.init(this)
         RxJavaPlugins.setErrorHandler(Timber::e)
-        Reaktor.handleErrorsWith(handler = Timber::e)
+        Reaktor.handleErrorsWith(crashInDebug = true, handler = Timber::e)
```

Comment:
as @FlowMo7 mentioned, probably use the AppInfo debug field to link with debug

@floschu in the Reaktor
`private val shouldCrash: Boolean
        get() = Reaktor.crashInDebug && BuildConfig.DEBUG`

only works with BuildConfig flag which might contradicts with the AppInfo debug

Reply:
> Another (more reaktor) thing:
> The `BuildConfig` class used within the Reaktor class is imported from the reaktor, right?
Yes, you're right.

Reply:
Please raise this (valid) issue in the reaktor repository